services/ui-src/src/measures/2022/CommonQuestions/OptionalMeasureStrat/ndrSets.tsx

```diff
@@ -453,7 +516,7 @@ const useRenderOPMCheckboxOptions = (name: string) => {
           >
             {customPrompt ??
               `Enter a number for the numerator and the denominator. Rate will
-auto-calculate:`}
+            auto-calculate`}
```

Comment:
Total nit, but do we need to keep this colon?
```suggestion
            auto-calculate:`}
```